### PR TITLE
In Sync Replicas metric displays correct information

### PR DESCRIPTION
**Pull request by ssiradeghyan**
- [ ] **Breaking change?** (if so, please describe the impact and migration path for existing application instances)

**What changes did you make?** (Give an overview)
fix Invalid "in sync replicas" value is displayed 

**Is there anything you'd like reviewers to focus on?**

**How Has This Been Tested?** (put an "x" (case-sensitive!) next to an item)

- [ ] No need to
- [x] Manually (please, describe, if necessary)
- [ ] Unit checks
- [ ] Integration checks
- [ ] Covered by existing automation

**Checklist** (put an "x" (case-sensitive!) next to all the items, otherwise the build will fail)
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (e.g. **ENVIRONMENT VARIABLES**)
- [x] My changes generate no new warnings (e.g. Sonar is happy)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged

Check out [Contributing](https://github.com/provectus/kafka-ui/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/provectus/kafka-ui/blob/master/CODE-OF-CONDUCT.md)

**A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

**Comment by workshur** [comment]
Please cover this case with tests

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "name": "kafka-ui",
+  "lockfileVersion": 2,
+  "requires": true,
+  "packages": {}
+}
```
Comment (workshur) [inline package-lock.json:6]: must be removed

**Review by workshur — APPROVED** [review]

**Review by Mgrdich — APPROVED** [review]

File: kafka-ui-react-app/src/components/Brokers/__test__/Brokers.spec.tsx [hunk 2]
```diff
@@ -67,5 +67,51 @@ describe('Brokers Component', () => {
...
+        outOfSyncReplicasCount: 1,
+      });
+      renderComponent();
+      await waitFor(() => {
+        expect(fetchStatsMock.called()).toBeTruthy();
+      });
+
+      const onlineWidgetDef = screen.getByText('798');
+      const onlineWidget = screen.getByText('of 799');
+      // const onlineIcon = screen.getAllByText()
```
Comment (workshur) [inline kafka-ui-react-app/src/components/Brokers/__test__/Brokers.spec.tsx:83]: pls revome unused code

File: kafka-ui-react-app/src/components/Brokers/__test__/Brokers.spec.tsx [hunk 3]
```diff
@@ -67,5 +67,51 @@ describe('Brokers Component', () => {
...
+        inSyncReplicasCount: 798,
+        outOfSyncReplicasCount: 1,
+      });
+      renderComponent();
+      await waitFor(() => {
+        expect(fetchStatsMock.called()).toBeTruthy();
+      });
+
+      const onlineWidgetDef = screen.getByText('798');
+      const onlineWidget = screen.getByText('of 799');
```
Comment (workshur) [inline kafka-ui-react-app/src/components/Brokers/__test__/Brokers.spec.tsx:82]: I see no need in creation of extra constants here. Pls use 
```js
expect(screen.getByText('798')).toBeInTheDocument();
expect(screen.getByText('of 799')).toBeInTheDocument();
```

File: kafka-ui-react-app/src/components/Brokers/__test__/Brokers.spec.tsx [hunk 4]
```diff
@@ -67,5 +67,51 @@ describe('Brokers Component', () => {
...
+        inSyncReplicasCount: undefined,
+        outOfSyncReplicasCount: 1,
+      });
+      renderComponent();
+      await waitFor(() => {
+        expect(fetchStatsMock.called()).toBeTruthy();
+      });
+
+      const onlineWidget = screen.getByText('of 1');
+      expect(onlineWidget).toBeInTheDocument();
```
Comment (workshur) [inline kafka-ui-react-app/src/components/Brokers/__test__/Brokers.spec.tsx:99]: same here

File: kafka-ui-react-app/src/components/Brokers/__test__/Brokers.spec.tsx [hunk 5]
```diff
@@ -67,5 +67,51 @@ describe('Brokers Component', () => {
...
+        outOfSyncReplicasCount: undefined,
+      });
+      renderComponent();
+      await waitFor(() => {
+        expect(fetchStatsMock.called()).toBeTruthy();
+      });
+      const onlineWidgetDef = screen.getByText('47');
+      const onlineWidget = screen.getByText('of 47');
+      expect(onlineWidgetDef).toBeInTheDocument();
+      expect(onlineWidget).toBeInTheDocument();
```
Comment (workshur) [inline kafka-ui-react-app/src/components/Brokers/__test__/Brokers.spec.tsx:114]: and here

**Review by workshur — APPROVED** [review]
@ssiradeghyan Please do the cleanup and merge the PR

**Comment by Mgrdich** [comment]
do we have to include the test report in it ?? can we just remove it.

File: kafka-ui-react-app/test-report.xml [hunk 6]
```diff
@@ -0,0 +1,883 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (workshur) [inline kafka-ui-react-app/test-report.xml:1]: Must be removed

**Pull request merged**